reviewed spanish expressions

mm-s opened the pull request.

jbrizio commented on locales/es.json:
```diff
@@ -2,9 +2,9 @@
 	"select-user-type.driver": "Conductor",
 	"select-user-type.passenger": "Pasajero",
-	"request-phone.type_your_phone": "Por favor envían tu número telefónico. No vamos a compartir tu información.",
+	"request-phone.type_your_phone": "Por favor envía tu número de telefóno. No vamos a compartir tu información.",
```
The correct spelling is `teléfono`

jbrizio commented on locales/es.json:
```diff
@@ -14,42 +14,42 @@
 	"order-submitted.order_submitted": "⏳ Localizando conductores... Si usted encontro un conductor, presione el botón de abajo 👇",
 	"order-submitted.cancel": "Terminar búsqueda",
-	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intente de nuevo.",
+	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intentelo de nuevo.",
```
The same case here. The correct spelling is `inténtelo`

asccigcc replied:
Guys maybe we should decide if the translation should be usted or tu. Warhammer do you think @jbrizio?

jbrizio replied:
I prefer to use _"usted"_

jbrizio commented on locales/es.json:
```diff
@@ -14,42 +14,42 @@
 	"driver-select-vehicle-type.car": "🚗 Automóvil",
 	"driver-select-vehicle-type.error_only_known_type": "Opción desconocida, por favor seleccione automóvil o motoneta / motocicleta.",
-	"driver-explain-checkins.text": "Casi terminamos, algunas cosas a tener en cuenta. Telegram no tiene soporte activo de localización GPS así que cada vez que cambie su ubicación realice un registro en la aplicación o registre cada hora si esta listo para aceptar nuevas ordenes de viaje. Esto puede ayudar a encontrar pasajeros cercanos a su ubicación. Presiona \"Siguiente\"",
+	"driver-explain-checkins.text": "Ya Casi hemos acabado, algunas cosas a tener en cuenta. Telegram no tiene soporte activo de localización GPS así que cada vez que cambie su ubicación realice un registro en la aplicación o registre cada hora si esta listo para aceptar nuevas ordenes de viaje. Esto puede ayudar a encontrar pasajeros cercanos a su ubicación. Presiona \"Siguiente\"",
```
Delete the Uppercase of `Casi`

jbrizio commented on locales/es.json:
```diff
@@ -14,42 +14,42 @@
-	"driver-request-location.provide_location": "Por favor comparta su ubicación actual, así los pasajeros cercanos a usted pueden encontrarle.",
-	"driver-explain-whats-next.text": "¡Estamos listos! Será notificado sobre pasajeros y ordenes cercanas a usted. Mantén esta aplicación activa y realice registros cada hora para actualizar su ubicación. Presione \"Siguiente\"",
+	"driver-request-location.provide_location": "Por favor comparta su ubicación actual, de modo que los pasajeros cercanos a usted puedan encontrarle.",
+	"driver-explain-whats-next.text": "¡listo! Será notificado sobre pasajeros y ordenes en su cercanos a usted. Mantenga esta aplicación activa y realice registros cada hora para actualizar su ubicación. Presione \"Siguiente\"",
```
I guess that the correct phrase should be:
`¡Listo! Usted será notificado sobre pasajeros y ordenes cercanas.`

mm-s replied:
Yes, sorry, I didn't review my review well..

ro31337 commented:
@mm-s do you agree with @jbrizio comments? If so, please add those fixes, or leave a comment here. Thanks

ro31337 commented:
Also @asccigcc what do you think?

asccigcc commented:
Agree, it is a typo. Good catch guys.

Dannymatos73 commented:
my location is 730 Richmond ave san Jose CA 95128 #3

On Feb 3, 2017 7:13 AM, "Jonathan Brizio" <notifications@github.com> wrote:

*@jbrizio* commented on this pull request.
------------------------------

In locales/es.json <https://github.com/ro31337/libretaxi/pull/437>:

> @@ -14,42 +14,42 @@
 	"passenger-request-destination.provide_destination": "¿A donde desea
ir? Por favor escriba la dirección exacta de la calle (Así el sistema
GPS podrá entender) o describa el lugar.",
 	"order-submitted.order_submitted": "⏳ Localizando conductores... Si
usted encontro un conductor, presione el botón de abajo 👇",
 	"order-submitted.cancel": "Terminar búsqueda",
-	"order-submitted.error_incorrect_input": "Opción desconocida, por
favor intente de nuevo.",
+	"order-submitted.error_incorrect_input": "Opción desconocida, por
favor intentelo de nuevo.",

I prefer to use *"usted"*

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/ro31337/libretaxi/pull/437>, or mute the thread
<https://github.com/notifications/unsubscribe-auth/AYUM0G_r0kaOYp4D9xQ93e4wWv20SQ22ks5rY0QTgaJpZM4Lytt_>
.

ro31337 commented:
Merged to master, feel free to add more improvements here, if any.

mm-s commented:
I've updated, there was a mixture of 'usted' and  'tu' usages, I've converted to 'tu' form as in Europe does not sound formal in excess, Latino America may have other preferences.

asccigcc commented:
Probably we should create es_MX and es_ES for each region. What do you think guys?
